### DISCUSSION TITLE
What handle dialog in cefsharp.dom?

### DISCUSSION THREAD

**Discussion by radiaku** [cefsharp/CefSharp.Dom#48] [Q&A]
My code is simple as this

```
var devToolsContext = await CefBrowser.ChromiumWebBrowser.CreateDevToolsContextAsync();
 foreach (var item in dataInput)
{
   await devToolsContext.GoToAsync("https://example.com/upload");
}
```
When I use it inside loop, its will show me dialog to **Reload**, and with yes or no.

in puppeter-sharp there [Dialog ](https://github.com/hardkoded/puppeteer-sharp/blob/dae5325e758e8b3cd75dbcec615a431893233d4b/lib/PuppeteerSharp/Dialog.cs)

but I found nothing on cefsharp.dom. or maybe I didnt see it. cause I cant searching on Cefsharp.dom github ( I really hope cefsharp.dom have own repository than result of forked one, its made easier to search. )

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/wiki/General-Usage#handlers

Start by trying to implement [IJsDialogHandler](https://cefsharp.github.io/api/110.0.x/html/T_CefSharp_IJsDialogHandler.htm)

**Reply by radiaku** [reply]
Yes, [IJsDialogHandler](https://cefsharp.github.io/api/110.0.x/html/T_CefSharp_IJsDialogHandler.htm) will do, but I do hope this will be on cefsharp.dom like puppeteersharp handle with dialog.
